__tests__/api.test.js

```diff
@@ -151,4 +152,49 @@ describe("/api/articles/:article_id/comments", () => {
         });
     });
   });
+  describe("POST", () => {
+    test("should return an object with user's comment with a status of 201", () => {
+      const testComment = {
+        user: "butter_bridge",
+        body: "Hello",
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(testComment)
+        .expect(201)
+        .then(({ body: { userComment } }) => {
+          expect(userComment).toEqual({
+            body: expect.any(String),
+          });
+        });
+    });
+    test("should return a status 400 with a message of bad request when article id is not a number", () => {
+      const testComment = {
+        user: "butter_bridge",
+        body: "Hello",
+      };
+      return request(app)
+        .post("/api/articles/hello/comments")
+        .send(testComment)
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("Bad request");
+        });
+    });
+    test("should return a status 422 with a message of unable to post comment to an article that does not exist", () => {
+      const testComment = {
+        user: "butter_bridge",
+        body: "Hello",
+      };
+      return request(app)
+        .post("/api/articles/444/comments")
+        .send(testComment)
+        .expect(422)
+        .then(({ body: { message } }) => {
+          expect(message).toBe(
+            "unable to post comment to an article that does not exist"
+          );
+        });
+    });
+  });
```

Comment:
We will need 2 more tests:
- 400: Bad request, incomplete req body (i.e. missing a key)
- 404: Not found, username does not exist (should be handled by SQL error)